.github/workflows/nightly.yml

```diff
@@ -89,6 +89,17 @@ jobs:
         if: matrix.build == 'linux' || matrix.build == 'macos'
         run: strip "target/release/seafowl"
 
+      - name: Test invoking the binaries
+        shell: bash
+        run: |
+          if [ "${{ matrix.os }}" = "windows-latest" ]; then
+            ./target/release/seafowl.exe --version
+            test $0 -eq 0 || exit $0
+          else
+            ./target/release/seafowl --version
+            test $0 -eq 0 || exit $0
```

Comment:
I think we should be using `$?` for the last exit status, right?

Alternatively, we could use the even shorter variant:
```bash
./target/release/seafowl --version || exit 1
```

Reply:
> I think we should be using `$?` for the last exit status, right?

Indeed! Thanks for catching my typo.

> Alternatively, we could use the even shorter variant:
> 
> ```shell
> ./target/release/seafowl --version || exit 1
> ```

I like this 👍 switched to it

Reply:
Awesome, thank you!